app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHours.kt

```diff
@@ -93,12 +93,25 @@ class AddOpeningHours(
             ),
         ).map { it.key + " ~ " + it.value.joinToString("|") }.joinToString("\n or ") + "\n" + """
             )
-            and !opening_hours
+            and (!opening_hours or opening_hours older today -1 years)
+          )
+          or
+          (
+          opening_hours older today -1 years
+          and
+            (
+            leisure=park
```

Comment:
I think that it is worth supporting (17 000 `leisure=park` with OH data, see http://overpass-turbo.eu/?q=LyoKVGhpcyBoYcSGYmVlbiBnxI1lcmF0ZWQgYnkgdGhlIG92xJJwxIlzLXR1cmJvIHdpemFyZC7EgsSdxJ9yaWdpbmFsIHNlxLBjaMSsxIk6CsOiwoDCnG9wxI3EumdfaG_EqHM9KiBhbsSXbGXEhcSoZT3Eo3JrxI9sb2LEvMWIwp0KKi8KW8WSdDpqc29uXVt0aW1lxbA6MjUwMF07Ci8vxI_ElMSdciByZXN1bHRzCigKICDGhiBxdcSSxJrFoXQgZm9yOiDFiMWKxYxuxY7FkMWScsWUxZbFmMWaxZzGj8aNxaDEsGvFqcaWIG5vZGVbIsWLxY1uxY_FkcWTIsW3IsWbxZ3GjSI9IsWha8eIxoTGl3dheceAx4LGqseExqzHh8eJx4vGtGXHjseQxrfHk8a6xo1sxJRpxbXHmcapxqvHhsaux4jHgMegxZ7Ho8eRx6YpxoTGmXDEt27GoMaNxo_GkcaTxbAgY8WSx747&c=BOyB35NeIQ

Reply:
Is it worth adding that to the CheckOpeningHoursSigned quest too (possibly one for another PR)?

Although would it make sense to extract the list of things which might have opening hours into a common list?